src/components/CartView.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+const CartView = ({ cart }) => {
+  let cartTotal = 0;
```

Comment:
see that's the variable, not a state. So its update wont trigger rerender, only state updates do. I believe it will display correct basket on initial render, but what if you add new item? I am not sure if the total price will be updated on the screen.

Reply:
I could be wrong here, but as it is only possible to add a product to the cart when the cart view is closed, so if you add a product then view your cart you will have the click the view cart button, which will cause it to rerender. This is how our test is set up at the moment. 
You add the first product, which creates the cart, then the cart is viewed and tests are run. The cart is closed, another product added, the cart view is opened again and it now displays two products with the combined price